features/admin.users.v1/components/user-profile.tsx

```diff
@@ -1070,6 +1073,17 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
                                     opValue = {
                                         [schemaNames[0]]: attributeValues
                                     };
+                                } else if (schema.extended) {
+                                    opValue = {
+                                        [schema.schemaId]: constructPatchOpValueForMultiValuedAttribute(
+                                            schema.name,
+                                            multiValuedAttributeValues[schema.name],
+                                            multiValuedInputFieldValue[schema.name]
+                                        )
+                                    };
+                                } else {
```

Comment:
No need of the else block. Just the comment is enough

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/8027/commits/913fa68968a319095b93afaa24e8f71a1a7642a6